HiP-WebserviceLib.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <RootNamespace>PaderbornUniversity.SILab.Hip.Webservice</RootNamespace>
     <NoWarn>1701;1702;1705;1591</NoWarn>
-    <Version>3.0.2</Version>
+    <Version>3.0.3</Version>
```

Comment:
Version should be 3.1.0 because this PR implements a new feature, not just a bugfix